goldstone/core/models.py

```diff
@@ -2696,6 +2701,69 @@ def build_alert_template(self, hits):
         return msg_params_dict
 
 
+class AlertSearchSQLQuery(AlertSearch):
+
+    db_table = models.CharField(max_length=64,
+                                default='compliance_vulnerability',
+                                help_text='database to be searched')
+
+    db_col_filter = models.CharField(max_length=64, default='created',
+                                     help_text='db column filter')
+
+    class Meta:               # pylint: disable=C0111,W0232,C1001
+
+        verbose_name_plural = "saved searches on DB data with alerts"
+
+    def search(self):
+        # This function returns a dictionary of rows (and not a list of rows)
+        # of all the entries in the vulnerability table.
+        # dict is more useful for k,v parsing. If you care about just a list
+        # simply return cursor.fetchall()
+        try:
+            cursor = connection.cursor()
+            cursor.execute("SELECT * FROM %(table)s",
+                           {"table": AsIs(self.db_table)})
+            columns = [col[0] for col in cursor.description]
+            return [dict(zip(columns, row))
```

Comment:
yes. the cursor also has a method that returns the number of hits. probably more efficient than processing everything into a list.